src/uucore/src/lib/features/entries.rs

```diff
@@ -72,6 +72,41 @@ pub fn get_groups() -> IOResult<Vec<gid_t>> {
     }
 }
 
+/// The list of group IDs returned from GNU's `groups` and GNU's `id --groups`
+/// starts with the effective group ID (egid).
+/// This is a wrapper for `get_groups()` to mimic this behavior.
+///
+/// If `arg_id` is None (default), `get_groups_gnu` moves the effective
```

Comment:
```suggestion
/// If `arg_id` is `None` (default), `get_groups_gnu` moves the effective
```
Might as well get really nitpicky :)